contracts/contracts/ExampleWarp.sol

```diff
@@ -36,7 +35,7 @@ contract ExampleWarp {
     }
 
     // validateGetBlockchainID checks that the blockchainID returned by warp matches the argument
-    function validateGetBlockchainID(blockchainID bytes32) external {
+    function validateGetBlockchainID(bytes32 blockchainID) external view {
```

Comment:
@aaronbuchwald, what would you do without @morrisettjohn?

Reply:
I just hope he doesn't go back to school 😅. We can give him a degree right?

Reply:
Haha that would be cool!